submissions/akulahere/DOM model/src/scripts/main.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const galleryList = document.querySelector('.gallery_list');
+const bigImage = document.querySelector('.gallery_large-img');
+const title = document.querySelector('figcaption');
+
+title.textContent = "Day and night"
```

Comment:
It is better not to mix single quotes and double quotes. Usually single quotes are for JS and double quotes for html. But most important is to be consistent.

Reply:
Fixed